ui/context_processors.py

```diff
@@ -15,7 +15,7 @@ def api_keys(request):
         "APIKEYS": {
             "GOOGLE": settings.GOOGLE_API_KEY,
             "GOOGLE_ANALYTICS": settings.GA_TRACKING_ID,
+            "GOOGLE_ADWORDS": settings.ADWORDS_CONVERSION_ID,
```

Comment:
should not we change ``settings.ADWORDS_CONVERSION_ID`` to ``GOOGLE_ADWORDS`` as well?